manifests/logical_volume.pp

```diff
@@ -6,10 +6,17 @@
   $fs_type           = 'ext4',
   $mountpath         = "/${name}",
   $mountpath_require = false,
+  # pass needs to be a parameter for managing root (/) volume
+  $pass              = '2'
 ) {
   validate_bool($mountpath_require)
 
   if $mountpath_require {
+    # Without this file mountpath resource, catalog runs fail for us on CentOS 6.5
+    file { "$mountpath":
+      ensure => directory,
+    }
```

Comment:
`$mountpath` should not be managed in the module. You should manage that in your node definition/role/profile.